src/metabase/api/card.clj

```diff
@@ -245,31 +244,19 @@
                       (when (instance? Throwable required-perms)
                         required-perms))))))
 
-(defn- save-new-card-async!
-  "Save `card-data` as a new Card on a separate thread. Returns a channel to fetch the response; closing this channel
-  will cancel the save."
-  [card-data user]
-  (async.u/cancelable-thread
-    (let [card (db/transaction
-                 ;; Adding a new card at `collection_position` could cause other cards in this
-                 ;; collection to change position, check that and fix it if needed
-                 (api/maybe-reconcile-collection-position! card-data)
-                 (db/insert! Card card-data))]
-      (events/publish-event! :card-create card)
-      ;; include same information returned by GET /api/card/:id since frontend replaces the Card it
-      ;; currently has with returned one -- See #4283
-      (-> card
-          (hydrate :creator
-                   :dashboard_count
-                   :can_write
-                   :average_query_time
-                   :last_query_start
-                   :collection [:moderation_reviews :moderator_details])
-          (assoc :last-edit-info (last-edit/edit-information-for-user user))))))
-
-(defn- create-card-async!
-  "Create a new Card asynchronously. Returns a channel for fetching the newly created Card, or an Exception if one was
-  thrown. Closing this channel before it finishes will cancel the Card creation."
+(def ^:private metadata-sync-wait-ms
+  "Duration in milliseconds to wait for the metadata before saving the card without the metadata. That metadata will be
+saved later when it is ready."
+  1500)
+
+(def ^:private metadata-async-timeout
```

Comment:
```suggestion
(def ^:private metadata-async-timeout-ms
```

Probably better to error on the side of overly explicit and make sure the fact that this is in milliseconds obvious just by looking at it. I know it's probably safe to assume it's in milliseconds since we didn't assume otherwise but I don't want someone who doesn't know that think this is in seconds and turn this into a 15,000 minute timeout